### ISSUE TITLE
The color of the underline is different from the text color.

### ISSUE THREAD

**Issue by m-yukio** [cocos2d/cocos2d-x#15214]
If use TrueType font, the color of the underline is different from the text color.

``` c++
Label* label = Label::createWithTTF("CHECK!", "ipaexm.ttf", 48.0f);
label->enableUnderline();
label->setColor(cocos2d::Color3B::BLUE);
label->setPosition(200.0, 400.0);
this->addChild(label);
label = Label::createWithSystemFont("CHECK!", "Verdana", 48.0f);
label->enableUnderline();
label->setColor(cocos2d::Color3B::BLUE);
label->setPosition(200.0, 300.0);
this->addChild(label);
```

![uilabel](https://cloud.githubusercontent.com/assets/9098173/13622886/8d41cf72-e5e5-11e5-87ba-ac3f82c4a790.jpg)

**Comment by ricardoquesada** [comment]
either wait until v3.13 is released, or use `setTextColor()` instead. thanks.

**Comment by ricardoquesada** [comment]
re-fixed in v3.14.

`setColor()` changes the color of the text and underline.
`setTextColor()` changes the color of only the text. This is the same behavior as system fonts.
